lib/linter.js

```diff
@@ -241,6 +231,47 @@ class Linter {
     };
   }
 
+  /**
+   * @param {Object} options
+   * @param {string} options.source - The source code to verify.
+   * @param {string} options.filePath
+   * @param {string} options.moduleId
+   * @param {Object[]} results - The lint messages.
+   *
+   * @returns {string} output - The fixed source.
+   */
+  _applyFixes(options, results) {
+    let fixableIssues = results.filter(r => r.isFixable);
```

Comment:
Do you prefer to have a `isFixable` property or checking against the existence of `fixer` function is enough?

Reply:
In this implementation, there is no fixer function (in the results listing, or in the rule instances). The idea here is to have _something_ in the results that the `ember-template-lint` command can use to add a bit to our output (like eslint itself has) for):

```
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
The different implementations were a bit mixed up in my mind.